web-admin/src/routes/[organization]/[project]/+layout.ts

```diff
@@ -0,0 +1,29 @@
+import { hasBlockerIssues } from "@rilldata/web-admin/features/billing/selectors";
+import { fetchAllProjectsHibernating } from "@rilldata/web-admin/features/organizations/selectors";
+import { error, redirect } from "@sveltejs/kit";
+
+export const load = async ({ params: { organization }, parent }) => {
+  const { organizationPermissions, issues } = await parent();
+  if (!organizationPermissions.manageOrg) {
+    return;
+  }
+
+  let shouldRedirectToProjectsList = false;
+
+  try {
+    // if all projects were hibernated due to a blocker issue on org then take the user to projects page
+    if (
+      hasBlockerIssues(issues) &&
+      (await fetchAllProjectsHibernating(organization))
+    ) {
+      shouldRedirectToProjectsList = true;
```

Comment:
It doesn't look like we need to keep this `shouldRedirectToProjectsList` state & we can just fire the redirect right here

Reply:
Oh I see. That's a little confusing, so we could only do the data fetching within the `try`, and move the conditional to the bottom?